lib/diagnostic.rb

```diff
@@ -115,10 +121,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = '{ :age => 25, :first_name => "Misha" }'
```

Comment:
Correct. You could also have it the other way

`{age: 25, first_name: 'Misha'}`